README.md

```diff
@@ -1,4 +1,7 @@
 # Kadena & Kadena.js
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
 
```

Comment:
Can we move this section below the short description? The first few lines are indexed for previews in various external tools. For the same reason I moved the Kadena logo down, as in previews we only displayed a bunch of html 😅

Reply:
Yes, we can do that after we merge this PR as this is generated by the bot.